tests/test_agent_manager.py

```diff
@@ -313,6 +318,22 @@ async def test_api(init_dataclasses_and_load_schema):
     assert_equal_ish(shouldbe, all_agents, sortby=["hostname", "name"])
     agentid = all_agents["processes"][0]["sid"]
 
+    start = agentid
+    code, all_agents_processes = await am.list_agent_processes(None, None, start=start)
```

Comment:
Changing these call to kwargs will make the code a lot more readable.

Reply:
I don't understand what you mean by that

Reply:
```suggestion
    code, all_agents_processes = await am.list_agent_processes(environment=None, expired=None, start=start)
```